### DISCUSSION TITLE
Welcome to PrestaShop GitHub Discussions!

### DISCUSSION THREAD

**Discussion by kpodemski** [PrestaShop/PrestaShop#30835] [Announcements]
GitHub Discussions are now enabled for the PrestaShop Project 🎉  We believe this feature will provide a more organized and easily accessible way for community members to collaborate, ask questions, and share ideas.

Read the full announcement [here](https://build.prestashop-project.org/news/2023/github-discussions-are-now-enabled-in-the-prestashop-project/).

**Comment by oroffini** [comment]
Hello @kpodemski does this mean that the Slack channel no longer exists?

**Reply by kpodemski** [reply]
Hello @oroffini 

It exists, https://www.prestashop-project.org/slack/

**Reply by oroffini** [reply]
Thank you, but i have this message: _"This link is no longer active.To join this workspace, you will need to ask the person who invited you to provide you with a new link."_

**Reply by kpodemski** [reply]
@oroffini sorry for that! could you try [this link](https://join.slack.com/t/prestashop/shared_invite/zt-2f9o67wg6-DgsDM2tT8vZvHjyIK614lQ)?

**Reply by kpodemski** [reply]
@oroffini I fixed a link on the website, thanks for letting us know!

**Reply by oroffini** [reply]
Yes It work perfectly. 
Thank you so much. 😀

**Comment by royanetyk** [comment]
Hello, when will we upgrade beyond PHP 8.1 because OVH no longer supports it, as in the photo with the red triangle! I wouldn't want to be vulnerable.
![Capture d’écran 2025-04-14 140521](https://github.com/user-attachments/assets/42293b38-bc72-4a07-a6f0-006d6214b70a)

**Reply by kpodemski** [reply]
Hello @royanetyk 

Security support for PHP 8.1 ends on December 31. You can consider upgrading your PrestaShop store to version 9 once it is released (somewhere in May), as PrestaShop 9 supports up to PHP 8.4.
